tests/data/test_10X_indexes

```diff
@@ -1,2 +1,3 @@
-SI-GA-A1,GGTTTACT,CTAAACGG,TCGGCGTC,AACCGTAA
+SI-GA-A1,CGCGCAG,CTAAACGG,TCGGCGTC,AACCGTAA
```

Comment:
I understand that it is easier to change the sequences of `SI-GA-A1` here. But it is a fixed index by 10X. Could it be better to change CGCGCAG into GGTTTACT instead?

Reply:
Ah, makes sense. I'll change it back then. Might have to revisit the tests that use this later on and improve their coverage in some other way then, but I'll leave that for now.